docs/userguide/howtouseicu.md

```diff
@@ -220,5 +220,5 @@ Any additional arguments will be passed to the program.
 
 *   `icurun test1.cpp *args...*`
 
-*This feature is a work in progress. Please give feedback at [Ticket
-#8481](https://unicode-org.atlassian.net/browse/ICU-8481).*
+*This feature is a work in progress. Please give feedback at
```

Comment:
FWIW, this ticket is closed as "Done" and there is nobody watching it -- so if somebody did give feedback/comments on the ticket nobody would notice. :(
I think it might be better to say something like:
"Please give feedback to the `icu-support` mailing list, and refer to Ticket #8481..." instead.

Reply:
Make sense. Done.

Reply:
thanks!